layers/+emacs/org/README.org

```diff
@@ -125,28 +126,54 @@ To enable notifications for agenda events, set the variable
-=TODO= headlines that include a timestamp (configuration options are documented
-[[https://github.com/akhramov/org-wild-notifier.el#configuration][here]]). The default configuration shows a notification 10 minutes before the
-event using the [[https://melpa.org/#/alert][alert]] package. The default behavior can be configured via the
-=org-wild-notifier-alert-time= variable. The notification type can be configured
-via the [[https://github.com/jwiegley/alert#for-users][the alert package its configuration settings]]. Additional notification
-times can be added using the [[https://github.com/akhramov/org-wild-notifier.el#configuration][:WILD_NOTIFIER_NOTIFY_BEFORE:]] keyword (*note* that
-[[https://github.com/akhramov/org-wild-notifier.el/issues/46][this additionally requires a =SCHEDULED:= stamp)]]. Properties can be added with
-=, i p=.
+timestamp to the headline (using =, d t= requires the =diary-file= variable to
+point to an existing file, see [[Agenda recommendations]]. You can also use the
+=org-journal-new-scheduled-entry= that becomes available by enabling [[Org-journal support][org-journal
+support]]). By default, the =org-wild-notifier= package will scan your
+=org-agenda-files= (every minute) for =TODO= headlines that include a timestamp
+(configuration options are documented [[https://github.com/akhramov/org-wild-notifier.el#configuration][here]]). The default configuration shows a
+notification 10 minutes before the event using the [[https://melpa.org/#/alert][alert]] package. The default
+behavior can be configured via the =org-wild-notifier-alert-time= variable. The
+notification type can be configured via the [[https://github.com/jwiegley/alert#for-users][the alert package its configuration
+settings]]. Additional notification times can be added using the
+[[https://github.com/akhramov/org-wild-notifier.el#configuration][:WILD_NOTIFIER_NOTIFY_BEFORE:]] keyword (*note* that [[https://github.com/akhramov/org-wild-notifier.el/issues/46][this additionally requires a
+=SCHEDULED:= stamp)]]. Properties can be added with =, i p=.
+
+** Agenda recommendations
+Emacs comes with its own diary package included. It is recommended not to use
+the diary directly. Instead of [[https://www.gnu.org/software/emacs/manual/html_node/emacs/Format-of-Diary-File.html][creating a diary file]], create a =diary.org=
+file and set the =diary-file= variable to point to that file. To make the
+holidays, birthdays etc. from the diary show up in the agenda, add the
+diary-style expression entries to your =diary.org= file as shown in the code
+block of the =Calendar/Diary integration= section [[https://orgmode.org/manual/Weekly_002fdaily-agenda.html][here]].
+
+Subsequently, to add appointments to the agenda you could use the following
+syntax:
+#+begin_src org
+  ,* Friday, 04/02/2021
+    ,** Appointment 1
+    <2021-04-02 Fri 10:10> 
+    ,** TODO Appointment 2
+    <2021-04-02 Fri 10:11>
+#+end_src
```

Comment:
Can you double check that org snippet? I am just wondering why most of the lines start with a `,` I would expect something like this:
``` org
    * Friday, 04/02/2021
    ** Appointment 1
    <2021-04-02 Fri 10:10> 
    ** TODO Appointment 2
    <2021-04-02 Fri 10:11>
```

Reply:
It is not possible to use the "normal" syntax in code blocks, it would just be recognized as being part of the org document its outline (which indeed happened without those comma's). This behavior and the `,` prefix solution is documented [here](https://orgmode.org/manual/Literal-Examples.html).

Reply:
TIL

Reply:
TIL me too... I had to look up its meaning

Reply:
Lol and just another kind of obscurity within the emacs world :), TIL indeed.